codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/transformers/EventStreamNormalizer.kt

```diff
@@ -37,3 +80,40 @@ object EventStreamNormalizer {
             .build()
     }
 }
+
+fun OperationShape.operationErrors(model: Model): List<Shape> {
```

Comment:
What is the intent for this function? The `OperationIndex.getErrors` already returns all the errors for the operation, right? So why does it need to be filtered by the synthetic trait? Wouldn't all of them have it since the normalizer added it to all of them?